packages/web-react/src/components/Dropdown/__tests__/useDropdownStyleProps.test.ts

```diff
@@ -1,35 +1,42 @@
 import { renderHook } from '@testing-library/react-hooks';
-import { DropdownPlacements } from '../../../types';
+import { PlacementDictionaryType } from '../../../types/shared/dictionaries';
```

Comment:
does the path have to be this specific? Couldn't we import it from the types only?

Reply:
Fixed.